examples/nomad-consul-separate-cluster/main.tf

```diff
@@ -84,9 +84,9 @@ module "nomad_servers" {
 
   # To make testing easier, we allow requests from any IP address here but in a production deployment, we strongly
   # recommend you limit this to the IP address ranges of known, trusted servers inside your VPC.
-  allowed_ssh_cidr_blocks = ["0.0.0.0/0"]
+  allowed_ssh_cidr_blocks = var.allowed_ssh_cidr_blocks
```

Comment:
Please see the comments above this and the READMEs: the code in the `examples` folder and the code in the root folder (e.g., `terraform-aws-nomad/main.tf`) is _example code_ that is intentionally left open for easy testing and learning. It is not meant to be used directly in production, and so we don't need the ability to configure the CIDR blocks here. The code that's meant to be used in production is in the `modules` folder, which already allows CIDR blocks to be configured (except in one place, which I'll comment on below).

Reply:
The idea to make the example configurable was born out of the idea to be able to not only test the happy path where SSH access is possible, but to be able to check the opposite as well. Sadly the bug in terraform contradicts this attempt.

Nevertheless I understand the intention to keep the example easy to be understood.

Would you prefer keeping the former version? With the changes proposed one might need to additionally check the `variables.tf` to understand the complete example. The current tests are not affected since a default is provided.

Reply:
Yea, prob better to revert the examples to the original approach to keep things simple and clear.